apps/cowswap-frontend/src/common/pure/NewModal/index.cosmos.tsx

```diff
@@ -0,0 +1,85 @@
+import ICON_ARROW from 'assets/icon/arrow.svg'
```

Comment:
NewModal? I would avoid naming it like this. Later we will have NewNewModal :) 

Or this is part of a plan of replacing the modals and it will be renamed later?